docs/local_testing.md

```diff
@@ -7,6 +7,11 @@ Run all tests other than Cypress in the project using `scripts/dev test`.
 - Run `scripts/dev test:go` to run all local-only server-side tests. This requires the database to be running first. Use `scripts/dev up:backend` to start it.
 - Run `scripts/dev test:go:only [full package name]` (e.g. `scripts/dev test:go:only "github.com/cmsgov/easi-app/pkg/cedar/core"`) to run server-side tests for a specific folder. Depending on the tests being run, this may require the database to be running, as above.
 - Run `scripts/dev test:go:long` to run all server-side tests, including ones that contact external services.
+- A single test method in a [`testify`](https://pkg.go.dev/github.com/stretchr/testify@v1.7.0) test suite can be run from the command line with `go test <package name> -testify.m <method name>`. This can be useful in cases where a test suite has many tests or if you just want to run a unit test without running tests that require external dependencies. Example:
+```
+go test "github.com/cmsgov/easi-app/pkg/services" -testify.m TestUpdateRejectionFields
+```
+This will run the `TestUpdateRejectionFields` method in [`pkg/services/system_intakes_test.go`](/pkg/services/system_intakes_test.go), which is part of the test suite for the entire `services` package.
 
```

Comment:
I also need to add some documentation on MailCatcher.

Reply:
Docs added in https://github.com/CMSgov/easi-app/pull/1543/commits/3dbc7be43a117422273672d656c97e3301c2647b, let me know what you think.